internal/cluster/connection_manager.go

```diff
@@ -306,6 +306,35 @@ func (m *ConnectionManager) RandomConnection() *Connection {
 	return m.connMap.RandomConn()
 }
 
+func (m *ConnectionManager) SQLConnection() *Connection {
+	if m.smartRouting {
+		if member := m.clusterService.SQLMember(); member != nil {
```

Comment:
There is a loop in the Java side (SQL_CONNECTION_RANDOM_ATTEMPTS times) with an explanation on top of it. Is this required here?

Reply:
We talked with @sancar about that. `SQL_CONNECTION_RANDOM_ATTEMPTS` doesn't make much sense. We should probably change other clients to remove that.